<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Merchandise_Search</class>
 <widget class="QDialog" name="Merchandise_Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search/Edit Merchandise</string>
  </property>
  <widget class="QPushButton" name="SearchMerchandise_BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchMerchandise_MerchandiseNumberBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchMerchandise_NumberSearch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumCommentsText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchMerchandise_MerchandiseKeyValidation">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="SearchMerchandise_MerchandiseDateBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchMerchandise_MerchandiseTitleBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_TitleText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Album Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumDateText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Publishing</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchMerchandise_TitleSearch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumNumberText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Album Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchMerchandise_MerchandiseTitleValidation">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumTitleText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Album Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchAlbums_SaveButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumKeyText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Words</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_NumberText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Album Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchMerchandise_MerchandiseCommentsValidation">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchMerchandise_SearchButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchMerchandise_MerchandiseCommentsBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchMerchandise_MerchandiseKeyBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SearchMerchandise_BackButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Search</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchMerchandise_SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Search</receiver>
   <slot>SearchMerchandise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchAlbums_SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Search</receiver>
   <slot>SaveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>SearchMerchandise()</slot>
  <slot>SaveChanges()</slot>
 </slots>
</ui>
